src/webui/www/private/index.html

```diff
@@ -7,7 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=10">
     <meta name="application-name" content="qBittorrent">
     <meta name="description" content="qBittorrent WebUI">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
```

Comment:
I'm concerned about this: https://stackoverflow.com/questions/2989263/disable-auto-zoom-in-input-text-tag-safari-on-iphone#comment90641872_46254706